giapi/giapi.h

```diff
@@ -107,6 +107,7 @@ namespace giapi {
 			PAUSE,
 			CONTINUE,
 			STOP,
+			STOP4,
```

Comment:
Why STOP4? What does the 4 stand for?

Reply:
It's `STOP-4` because it stops the observation after the last integration of a ABBA sequence. The normal STOP acts after the on-going integration is done, so at any point in the sequence.

Reply:
I changed the name of the new option to stopCycle